<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myselfinfoWidget</class>
 <widget class="QWidget" name="myselfinfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>321</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(165, 165, 165);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="myselfinfo_head_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid;
border-color: rgb(195, 195, 195);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../img.qrc">:/chat/man.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="myselfinfo_chatname_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小毛</string>
   </property>
  </widget>
  <widget class="QPushButton" name="myselfinfo_sendmsg_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(33, 150, 243);
border:0px;
border-radius:5px

</string>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid;
border-color: rgb(195, 195, 195);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="myselfinfo_userid">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="myselfinfo_address">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="myselfinfo_userid_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="myselfinfo_userid_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>地区：</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>myselfinfo_head_label</zorder>
  <zorder>myselfinfo_chatname_label</zorder>
  <zorder>myselfinfo_sendmsg_btn</zorder>
  <zorder>myselfinfo_userid</zorder>
  <zorder>myselfinfo_address</zorder>
  <zorder>myselfinfo_userid_2</zorder>
  <zorder>myselfinfo_userid_3</zorder>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
